src/client/app/components/about.ts

```diff
@@ -29,17 +37,62 @@ import { environment, storageSize, Strings, storage } from '../helpers';
     `
 })
 
-export class About {
+export class About implements AfterViewInit {
     @Input() show: boolean;
     @Output() showChange = new EventEmitter<boolean>();
+
+    cache = [
+        Strings.aboutStorage,
+        storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets),
+        storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)
+    ].join('\n');
+
     config = {
         build: environment.current.build,
         editorUrl: environment.current.config.editorUrl,
     };
 
-    cache = `
-    ${Strings.aboutStorage}
-    ${storageSize(localStorage, `playground_${environment.current.host}_snippets`, Strings.aboutSnippets)}
-    ${storageSize(sessionStorage, 'playground_intellisense', Strings.aboutIntellisense)}
-    `;
+    configs = [];
+
+    selectedConfig = '';
+
+    constructor(
+        private _effects: UIEffects,
+        private _changeDetector: ChangeDetectorRef
+    ) { }
+
+    ngAfterViewInit() {
+        this.configs.push(
+            { name: 'production', value: 'Production' },
+            { name: 'insiders', value: 'Beta' },
+            { name: 'edge', value: 'Alpha' },
+        );
+        if (environment.current.config.name === config['local'].name) {
+            this.configs.push({ name: 'local', value: config['local'].editorUrl });
+        }
+
+        this.selectedConfig = this.configs.find(c => c.name.toUpperCase() === environment.current.config.name).name;
+    }
+
+    async changeConfig(newConfig: string) {
+        if (config[newConfig] && config[newConfig].name !== environment.current.config.name) {
+            let currentConfigName = environment.current.config.name.toLowerCase();
+            let result = await this._effects.alert(
+                Strings.changeEnvironmentConfirm,
+                `${Strings.aboutSwitchEnvironment} ${this.configs.find(c => c.name === currentConfigName).value } to ${this.configs.find(c => c.name === this.selectedConfig).value }`,
+                Strings.okButtonLabel,
+                Strings.cancelButtonLabel
+            );
+
+            if (result === Strings.cancelButtonLabel) {
+                this.selectedConfig = this.configs.find(c => c.name === currentConfigName).name;
+                this._changeDetector.detectChanges();
```

Comment:
detectChanges may not be necessary if we use ng-change instead of change, but I am not an Angular expert